formver.compiler-plugin/core/src/org/jetbrains/kotlin/formver/core/conversion/ExactLocationException.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.kotlin.formver.core.conversion
+
+import org.jetbrains.kotlin.KtSourceElement
+
+class ExactLocationException(val source: KtSourceElement?, override val message: String) : Exception()
```

Comment:
Thinking about it some more, I'd recommend calling this `PluginInternalException`.  As it is, it sounds like this exception is thrown *because* there is an exact location, which isn't the case.  It is also not quite true, since `source` can be `null` (and besides, "exact" is a vague term).